pkg/controller/networkpolicy/clusternetworkpolicy.go

```diff
@@ -254,13 +257,157 @@ func (n *NetworkPolicyController) deleteNamespace(old interface{}) {
 	}
 }
 
+// filterACNPsByServiceAccount gets all ClusterNetworkPolicies that will need to be
+// re-processed based on if they are affected by an added/updated/deleted
+// ServiceAccount.
+func (n *NetworkPolicyController) filterACNPsByServiceAccount(sa *v1.ServiceAccount) (acnps []*crdv1alpha1.ClusterNetworkPolicy) {
+	acnpObjs, err := n.cnpInformer.Informer().GetIndexer().ByIndex(ServiceAccountsPeerIndex, HasServiceAccountsPeer)
```

Comment:
@tnqn is there any issue with defining an index like this which simply partitions all the objects into 2 different groups?